README.md

```diff
@@ -1,2 +1,31 @@
 # movie_recommender
 Recommendation Engine (PoC)
+
+## Files usage
+
+#### API-related files
+In order to run the recommendation engine in a micro-service, the following files are needed:
+* online_clustering.py (categories recommendator)
+* online_nmf.py (training for the user's content recommendator)
+* onlinereco.py (user's content recommendator)
+* reco_api.py (API)
+Along with the following files from the movielens dataset:
+* ratings.csv
+* movies.csv
+These csv files are not contained in this repository.
+
+
+If all of these files are present it is enough to run *reco_api.py* with python 2.7.15. inside the virtual environment contained in *environment.yaml*, which can be created by running `conda env create -f environment.yaml`.
```

Comment:
Please add a section like:

#### Setup
```
$ conda env create -f environment.yaml
```

#### Usage
```
$ python reco_api.py
```

and so on...

Reply:
Please also include a `curl` command specifying how to test the recommendation endpoints.